python/ray/dag/compiled_dag_node.py

```diff
@@ -774,9 +774,10 @@ def __init__(
                 be blocked in the first place; therefore, this limit is only
                 enforced when it is smaller than the DAG capacity.
             max_inflight_executions: The maximum number of in-flight executions that
-                are allowed to be sent to this DAG. Before submitting more requests,
-                the caller is responsible for calling ray.get to get the result,
-                otherwise, RayCgraphCapacityExceeded is raised.
+                can be submitted via `execute` or `execute_async` before consuming
+                the output using `ray.get()`. Before calling more `execute` or
+                `execute_async`, the caller is responsible for calling `ray.get()`
+                to get the result, otherwise, `RayCgraphCapacityExceeded` is raised.
```

Comment:
nit, this part seems repetitive
```suggestion
                the output using `ray.get()`. If the caller submits more executions,
                `RayCgraphCapacityExceeded` is raised.
```

Reply:
updated